manage.py

```diff
@@ -367,6 +370,93 @@ def index_advisory_opinions():
                 print("%d docs loaded" % docs_loaded)
         print("%d docs loaded" % docs_loaded)
 
+def get_xml_tree_from_url(url):
+    r = requests.get(url, stream=True)
+
+    with open('temp.zip', 'wb') as f:
+        for chunk in r:
+            f.write(chunk)
+    zip_file = ZipFile('temp.zip')
+
+    with zip_file.open(zip_file.namelist()[0]) as title,\
+     open('title.xml', 'wb') as title_xml:
+        title_xml.write(title.read())
+
+    return ET.parse('title.xml')
+
+def get_title_52_statutes():
+    es = utils.get_elasticsearch_connection()
+
+    title_parsed = get_xml_tree_from_url('http://uscode.house.gov/download/' +
+                    'releasepoints/us/pl/114/219/xml_usc52@114-219.zip')
+    tag_name = '{{http://xml.house.gov/schemas/uslm/1.0}}{0}'
+    for subtitle in title_parsed.iter(tag_name.format('subtitle')):
+        if subtitle.attrib['identifier'] == '/us/usc/t52/stIII':
+            for subchapter in subtitle.iter(tag_name.format('subchapter')):
+                match = re.match("/us/usc/t52/stIII/ch([0-9]+)/sch([IVX]+)",
```

Comment:
Not a huge deal, but I prefer to use symbolic group names in regex's for readability, as well as pre-compile the regexs for performance if I'm re-using them.